mozilla-release/toolkit/mozapps/extensions/content/extensions.js

```diff
@@ -1746,9 +1746,19 @@ var gCategories = {
     this.node = document.getElementById("categories");
     this._search = this.get("addons://search/");
 
+    // These addon categories are disabled in CLIQZ.
+    const disabledCategories = new Set([
```

Comment:
@maxim-cliqz can you please use "extensions.cliqz.listed" pref and only disable the extension page if that pref is false? Just like in  https://github.com/cliqz-oss/browser-f/blob/master/cliqz.cfg#L312